classes/class-wc-rest-connect-self-help-controller.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! defined( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH' ) ) {
+	define( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH', 32 );
```

Comment:
why not just re-use https://github.com/Automattic/woocommerce-connect-client/blob/9ca816e35286eaef4727fe9b64b6b6a3a61dc380/classes/class-wc-rest-connect-services-controller.php#L8 ?